project/SparkBuild.scala

```diff
@@ -599,11 +599,18 @@ object SparkParallelTestGrouping {
 
 object Core {
   import scala.sys.process.Process
+  def buildenv = Process(Seq("uname")).!!.trim.replaceFirst("[^A-Za-z0-9].*", "").toLowerCase
+  def bashpath = Process(Seq("where", "bash")).!!.split("[\r\n]+").head.replace('\\', '/')
```

Comment:
You could inline this where used below, but not sure it matters

Reply:
I originally had it inline, but it was not as readable, so I pulled it out.